README.md

```diff
@@ -3,4 +3,6 @@ Repo for general experiments
 
 # List of recipes containing broccoli
 * Penne with broccoli and cream
-    (don't forget the garlic)
+ (Try pink pepper next time)
```

Comment:
I think here it should say "Try pink pepper and garlic next time". Can you update, please?